BT-Tracking/Interface/Active/ActiveAppVC.swift

```diff
@@ -418,49 +418,63 @@ private extension ActiveAppVC {
     func debugProcessReports() {
         showProgress()
 
-        _ = viewModel.reporter.downloadKeys(lastProcessedFileName: nil) { [weak self] result in
-            switch result {
-            case .success(let keys):
-                self?.viewModel.exposureService.detectExposures(
-                    configuration: RemoteValues.exposureConfiguration,
-                    URLs: keys.URLs
-                ) { [weak self] result in
+        let configuration = RemoteValues.exposureConfiguration
+        let keyURLs = AppSettings.traveler ? RemoteValues.keyExportEuTravellerUrls : RemoteValues.keyExportNonTravellerUrls
+        _ = viewModel.reporter.downloadKeys(exportURLs: keyURLs, lastProcessedFileNames: [:]) { [weak self] report in
+            guard let self = self else { return }
+
+            let dispatchGroup = DispatchGroup()
+            var resultText: String = ""
+
+            for (code, success) in report.success {
+                dispatchGroup.enter()
+                self.viewModel.exposureService.detectExposures(configuration: configuration, URLs: success.URLs) { [weak self] result in
```

Comment:
Uz je to hodne necitelne. Muzes to vytahnout do vlastni fce a pojmenovat tak, aby to bylo co nejvic self-explaining.